src/structures/AutocompleteInteraction.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const BaseCommandInteraction = require('./BaseCommandInteraction');
+const CommandInteractionOptionResolver = require('./CommandInteractionOptionResolver');
+const { InteractionResponseTypes } = require('../util/Constants');
+
+/**
+ * Represents an autocomplete interaction.
+ * @extends {BaseCommandInteraction}
+ */
+class AutocompleteInteraction extends BaseCommandInteraction {
```

Comment:
I was suggesting to continue extending Interaction and duplicate the few necessary parameters and methods, since webhook, ephemeral, deferred (from BaseCommandInteraction) don't apply here